packages/pointer_interceptor/example/README.md

```diff
@@ -8,7 +8,7 @@ An example for the PointerInterceptor widget.
 
 ## Running tests
 
-`flutter drive --target integration_test/widget_test.dart --driver test_driver/integration_test.dart --show-web-server-device -d web-server`
+`flutter drive --target integration_test/widget_test.dart --driver test_driver/integration_test.dart --show-web-server-device -d web-server --web-renderer=html`
```

Comment:
I checked the tests again, and this is an OK change.

The test is written to inspect the HTML code of the page, and the way views are rendered is slightly different between the html and canvaskit renderers.

In this case, `html` is the way to go. I'll try to figure out if there's a way to get canvaskit coverage as well, so this can be tested with both renderers!